### ISSUE TITLE
Containerized Chrome, extension does not connect to keepass anymore

### ISSUE THREAD

**Issue by len-ro** [keepassxreboot/keepassxc-browser#1665]
## Expected Behavior
I have been running a docker-ized chrome version for a few years now with a shared keepass on the host without issues by mounting:

-v /run/user/$UID/org.keepassxc.KeePassXC.BrowserServer:/tmp/org.keepassxc.KeePassXC.BrowserServer 

(based on reading code in BrowserShared.cpp since in docker the xdgRuntimeDir is empty thus reverts to /tmp)

and copying the org.keepassxc.keepassxc_browser.json file to the proper location in the container

## Current Behavior

Since upgrading chrome/keepassxc-browser this does not work anymore. I am getting:

`Message encryption failed. Is KeePassXC running?` or `Key exchange was not successful`

I am tried to spend some time to do a strace to find if the socket is still used which seems to be the case. Have there been any notable changes which could explain this behaviour?

Thank you.

## Debug info
KeePassXC - 2.7.1
KeePassXC-Browser - 1.8.0
Operating system: Linux x86_64 (in docker)
Browser: Chrome/Chromium 103.0.0.0

**Comment by varjolintu** [comment]
This https://github.com/keepassxreboot/keepassxc/pull/8030. But it should only affect to 2.7.2.

**Comment by len-ro** [comment]
Thank you for this information. Apparently for some reason if the socket was in /tmp it was looking in /tmp/runtime-${USER} and if in /tmp/runtime-${USER} it was looking in /tmp. I changed my config to create the /run/user/$UID folder and mount it in docker and also export the XDG_RUNTIME_DIR to point to this folder and it seems to work now. I think this change occurred when I upgraded my container base from ubuntu 20.04 to 22.04.
Thank you for the help.
